evap/static/ts/src/notebook.ts

```diff
@@ -0,0 +1,70 @@
+declare const bootstrap: typeof import("bootstrap");
+import { assertDefinedUnwrap } from "./utils.js";
+import { CSRF_HEADERS } from "./csrf-utils.js";
+
+const NOTEBOOK_LOCALSTORAGE_KEY = "evap_notebook_open";
+
+if (localStorage.getItem(NOTEBOOK_LOCALSTORAGE_KEY) == "true") {
+    new bootstrap.Collapse(assertDefinedUnwrap(document.getElementById("notebook")));
+    new bootstrap.Collapse(assertDefinedUnwrap(document.getElementById("notebookButton")));
```

Comment:
What do these two lines do? I'm asking because I feel they enable some kind of functionality, and they should thus be outside the `if`-block.

Reply:
I think you said you wanted to post a link to some kind of explanation here...